scripts/check_enclave_reproducibility

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o xtrace
+
+# This script builds the Asylo server twice from scratch (cleaning the Bazel cache before each
+# compilation), saves the hash of the trusted enclave code to a temporary file, and compares the
+# hashes of the two artifacts.
+# It prints nothing if the hashes match, or a diff in case they don't match.
+
+bazel clean
+rm -rf ./bazel-cache/*
+./scripts/build_server
+./scripts/print_enclave_hash > /tmp/hash_0
```

Comment:
`$TMPDIR` rather than hardcoded `/tmp` ?

Reply:
`$TMPDIR` seems not to be set in our Docker image, not sure why.